VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -116,7 +123,7 @@ protected void onSuccess(CourseTopics courseTopics) throws Exception {
                     }
                 }
             };
-            getTopicsTask.setProgressCallback(null);
+            getTopicsTask.setProgressDialog(loadingIndicator);
```

Comment:
It looks like the progress bar is hidden from the `onFinally()` callback of `Task`, which is called _after_ `onSuccess()`, thus causing it to be potentially hidden immediately after it has been shown by the pagination loader initialization. The solution to that would be to have it manually managed here, and ensure that it's hidden _before_ the potential call to `populatePostListRunnable.run()`.